### DISCUSSION TITLE
v1.69.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#3401] [Announcements]
# v1.69.0

**Please make sure to use the latest `docker-compose.yml` in your environment to avoid start up issues**

Hello everybody, welcome to release `v1.69.0` of Immich. In this release, we are focusing on fixing bugs and improving some UI elements across the mobile app and the web client. Some of the key highlights include:

- Add support for Motion photos on Android.
- Improve UI for hide and show face mechanism.
- Fix a bug where files with uppercase extension cannot be uploaded from the web.
- Fix a bug where faces after the merging process don’t get recognized in new assets.

### Support Motion Photos
Thanks to the work of @fourkbomb, motion photos are now supported in Immich, the motion portion of the image is now extracted into a video file and can be played from both the web and the mobile app. Please give it a try and let us know if you encounter any issues.

### Improve hide/show faces user interface
After receiving a round of feedback on this mechanism from the last release, we’ve made improvements on the intuitiveness of the action of hiding and showing faces.

https://github.com/immich-app/immich/assets/74269598/97d6e798-803a-432e-9c06-797b4aca256f

- - - -

And as always, bugs are fixed, and many other improvements also come with this release.

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="Loading ~4000 images/videos"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

🎉 Cheers! 🎉

### Server
* fix(server): transcode live photos on upload by @mertalev in https://github.com/immich-app/immich/pull/3354
* fix(server): properly handle SIGTERM by @bt90 in https://github.com/immich-app/immich/pull/3350
* feat(server): Google Pixel motion photos by @fourkbomb in https://github.com/immich-app/immich/pull/3175
* fix(server): merged faces cannot be recognized in new photos by @alextran1502 in https://github.com/immich-app/immich/pull/3381

### Web
* fix(web): cannot upload file with uppercase extension by @alextran1502 in https://github.com/immich-app/immich/pull/3374
* fix(web): improve hide faces by @martabal in https://github.com/immich-app/immich/pull/3352

### Mobile
* feat(mobile): render login fields/buttons based on server configuration by @ddshd in https://github.com/immich-app/immich/pull/3339
* fix(mobile): AssetCount reset and Elliptical progress in Memories by @shalong-tanwen in https://github.com/immich-app/immich/pull/3355
* fix(mobile): Uses gray box placeholder for loading images by default and fixes odd spinner  by @martyfuhry in https://github.com/immich-app/immich/pull/3364
* feat(mobile): Precaches next image in memories by @martyfuhry in https://github.com/immich-app/immich/pull/3365
* fix(mobile): Fixes hide controls when zoomed and shows them when not zoomed by @martyfuhry in https://github.com/immich-app/immich/pull/3366
* fix(mobile): Decrease ScrollBar Fade timeout to a second by @shalong-tanwen in https://github.com/immich-app/immich/pull/3370

### Machine Learning
* chore(ml): move dev dependencies by @mertalev in https://github.com/immich-app/immich/pull/3379
* fix(ml): startup issue in dev by @alextran1502 in https://github.com/immich-app/immich/pull/3373

### CLI
* feat(cli): add build script and unify tests by @oddlama in https://github.com/immich-app/immich/pull/3369
* fix(cli): move runtime dependencies to correct section by @oddlama in https://github.com/immich-app/immich/pull/3371
* fix(cli): read-only and sidecar support for import by @alex-phillips in https://github.com/immich-app/immich/pull/3372

### Others
* chore(deps): bump stumpylog/image-cleaner-action from 0.1.0 to 0.2.0 by @dependabot in https://github.com/immich-app/immich/pull/3361

## New Contributors
* @fourkbomb made their first contribution in https://github.com/immich-app/immich/pull/3175
* @shalong-tanwen made their first contribution in https://github.com/immich-app/immich/pull/3355

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.68.0...v1.69.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.69.0'>v1.69.0</a>.</em>

**Comment by bt90** [comment]
While not one of the highlights, #3350 should finally get rid of the 10s it takes to stop or restart immich containers.

**Comment by wjbeckett** [comment]
I assume the Android Motion Photos feature will only support newly uploaded assets since the change? It can't look back over the existing photos and stitch the photo to the video?

**Reply by fourkbomb** [reply]
You need to rerun the "extract metadata" job for all photos. That should pick up any existing motion photos.

**Reply by bt90** [reply]
And "Transcode Videos" after that, I guess?

**Reply by fourkbomb** [reply]
I don't think so - the extracted videos should be treated as if they were newly uploaded (I may be wrong, though).

**Reply by bt90** [reply]
The metadata step yielded 518 missing transcodes on my instance.

**Comment by HiFallMaple** [comment]
Hello, I found that the "LivePhoto backup and playback" field in the Features section of the Github description only supports IOS, but in v1.69.0 it already supports Android Motion Photo.

However, Motion Photo for Android is now supported in this version. I'm a Samsung user and I'm very excited about this feature. I'm a Samsung user and I'm very excited about this feature. 

I was wondering if it's possible to change the Github description to support both Android and IOS? I think it would be more appealing!
